tests-integration/src/test_util.rs

```diff
@@ -338,6 +343,42 @@ pub async fn setup_test_http_app_with_frontend(
     (app, guard)
 }
 
+fn mock_insert_request() -> InsertRequest {
+    /***
+    let column_schemas = vec![
+        ColumnSchema::new("host", ConcreteDataType::string_datatype(), false),
+        ColumnSchema::new("cpu", ConcreteDataType::float64_datatype(), true),
+        ColumnSchema::new("memory", ConcreteDataType::float64_datatype(), true),
+        ColumnSchema::new("ts", ts_type, true).with_time_index(true),
+    ];
+    ***/
+
+    let mut columns_values = HashMap::with_capacity(4);
+    let mut builder = StringVectorBuilder::with_capacity(1);
+    builder.push(Some("host1"));
+    columns_values.insert("host".to_string(), builder.to_vector());
+
+    let mut builder = Float64VectorBuilder::with_capacity(1);
+    builder.push(Some(1.1));
+    columns_values.insert("cpu".to_string(), builder.to_vector());
+
+    let mut builder = Float64VectorBuilder::with_capacity(1);
+    builder.push(Some(2.0));
+    columns_values.insert("memory".to_string(), builder.to_vector());
+
+    let mut builder = TimestampSecondVectorBuilder::with_capacity(1);
```

Comment:
I find the cause. It's panic at https://github.com/GreptimeTeam/greptimedb/blob/c5f6d7c99ae25832603be90f0d09bfd20e3337f8/src/promql/src/extension_plan/normalize.rs#L225

Because we suppose the timestamp column type is always `TimestampMillisecond` in Prometheus tables. So the `downcase_ref` will return None and panic when `unwrap`.

Reply:
Thank you for helping me!! I'll write tests today or tomorrow.